62_CAD/vertex_shader.hlsl

```diff
@@ -7,6 +7,11 @@
 #include <nbl/builtin/hlsl/algorithm.hlsl>
 #include <nbl/builtin/hlsl/jit/device_capabilities.hlsl>
 
+using ActualFloat64 = portable_float64_t<jit::device_capabilities>;
```

Comment:
can you add a mode/macro "ForceEmulatedFloat64" for us to test on GPUs that already have Float64 feature?
or alternatively tell me how you usually test emulated vs non-emulated rapidly?

Reply:
https://github.com/Devsh-Graphics-Programming/Nabla/blob/emulated_float64_t/include/nbl/builtin/hlsl/portable/float64_t.hlsl#L13
```
// force emulated_float64<true, true>
using portable_float64_t = typename conditional<device_capabilities_traits<device_caps>::shaderFloat64, emulated_float64_t<true, true>, emulated_float64_t<true, true> >::type;

// force emulated_float64<true, true>
using portable_float64_t = typename conditional<device_capabilities_traits<device_caps>::shaderFloat64, float64_t, emulated_float64_t<true, true> >::type;
```

I actually forgot to change that back, so emulated float is forced

Reply:
then I think you can do this `ForceEmulatedFloat64` option by using a fake caps struct

Reply:
```cpp
#ifdef NBL_FORCE_EMULATED_FLOAT_64
using portable_float64_t = emulated_float64_t<true, true>;
#else
using portable_float64_t = typename conditional<device_capabilities_traits<device_caps>::shaderFloat64, float64_t, emulated_float64_t<true, true> >::type;
#endif
```
did this, so in order to force ef64 one has to define `NBL_FORCE_EMULATED_FLOAT_64` macro before including `portable/float64.hlsl`